lib/_stream_readable.js

```diff
@@ -644,11 +644,8 @@ function maybeReadMore_(stream, state) {
   state.readingMore = false;
 }
 
-// Abstract method.  to be overridden in specific implementation classes.
-// call cb(er, data) where data is <= n in length.
-// for virtual (non-string, non-buffer) streams, "length" is somewhat
-// arbitrary, and perhaps not very meaningful.
-Readable.prototype._read = function(n) {
+// Abstract method. To be overridden in specific implementation classes.
+Readable.prototype._read = function() {
```

Comment:
Can you please restore `n` here. It's meant to signal the need to have an `n` argument to the reader. Also, the previous comment was important information.

Reply:
I wasn't ware aware of it. But I haven't seen `n` used inside the `_read` method. So what this `n` does basically?

Reply:
Ok I have reverted it back. But still I'm unclear why the unused variable should be present. Only the comment/docs should meant the intention right?

Reply:
The error points to a line of code that can be inspected. In this way the reader can copy paste it.